pkg/sink/sink_test.go

```diff
@@ -920,173 +916,6 @@ func TestHandleEvent(t *testing.T) {
 	}
 }
 
-// Setup for TestHandleEvent_AuthOverride
-const userWithPermissions = "user-with-permissions"
-const userWithoutPermissions = "user-with-no-permissions"
-const userWithForbiddenAccess = "user-forbidden"
-
-var triggerAuthWG sync.WaitGroup
-
-type fakeAuth struct {
-}
-
-func (r fakeAuth) OverrideAuthentication(sa string, _ string, _ *zap.SugaredLogger, defaultDiscoverClient discoveryclient.ServerResourcesInterface,
```

Comment:
FYI @gabemontero @khrm With this change, we are no longer returning 4xx errors for unauthorized resource creation. Does this impact anything you are working on?

Reply:
No impacts on my end.